app/src/main/java/com/orange/ouds/app/ui/tokens/TokenCategory.kt

```diff
@@ -96,32 +114,83 @@ sealed class TokenCategory(
         }
     }
 
-    data object Elevation : TokenCategory(
-        R.string.app_tokens_elevation_label,
-        R.drawable.ic_layers,
-        R.string.app_tokens_elevation_description_text
+    data object Elevation : TokenProperty(
+        nameRes = null,
+        tokens = { OudsElevationKeyToken.entries.map { Token(it.name, it.value) } }
     ) {
         @Composable
         fun Illustration(elevation: Dp) {
             Surface(shadowElevation = elevation) {
                 Box(
                     modifier = Modifier
-                        .size(64.dp)
+                        .size(defaultIllustrationSize)
                         .background(color = OudsColorKeyToken.Surface.value), //TODO use BgDefaultSecondary token when available
                 )
             }
         }
     }
 
+    data object BorderRadius : TokenProperty(
+        nameRes = R.string.app_tokens_border_radius_label,
+        tokens = { OudsBorderRadiusKeyToken.entries.map { Token(it.name, it.value) } }
+    ) {
+        @Composable
+        fun Illustration(radius: Dp) {
+            Box(
+                modifier = Modifier
+                    .size(defaultIllustrationSize)
+                    .border(
+                        width = 1.dp,
+                        color = OudsColorKeyToken.OnSurface.value,
+                        shape = RoundedCornerShape(radius)
+                    ) //TODO use ContentDefault token when available
+                    .background(color = OudsColorKeyToken.Surface.value), //TODO use BgDefaultSecondary token when available
+            )
+        }
+    }
+
+    data object BorderWidth : TokenProperty(
+        nameRes = R.string.app_tokens_border_width_label,
+        tokens = { OudsBorderWidthKeyToken.entries.map { Token(it.name, it.value) } }
+    ) {
+        @Composable
+        fun Illustration(width: Dp) {
+            Box(
+                modifier = Modifier
+                    .size(defaultIllustrationSize)
+                    .border(width = width, color = OudsColorKeyToken.OnSurface.value) //TODO use ContentDefault token when available
+                    .background(color = OudsColorKeyToken.Surface.value), //TODO use BgDefaultSecondary token when available
+            )
+        }
+    }
+
+    data object BorderStyle : TokenProperty(
+        nameRes = R.string.app_tokens_border_style_label,
+        tokens = { OudsBorderStyleKeyToken.entries.map { Token(it.name, it.value) } }
+    ) {
+        @Composable
+        fun Illustration(style: OudsBorderStyle) {
+            val modifier = when (style) {
+                OudsBorderStyle.None -> Modifier
+                OudsBorderStyle.Solid -> Modifier.border(width = 1.dp, color = OudsColorKeyToken.OnSurface.value) //TODO use ContentDefault token when available
+                OudsBorderStyle.Dashed -> Modifier.dashedBorder(color = OudsColorKeyToken.OnSurface.value, shape = RectangleShape)
```

Comment:
It's a detail because we will probably not forget it, but `TODO` comment is missing for this line.